src/components/PersonLink.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Person } from '../types';
+
+interface Props {
+  person: Person;
+  activeSlug: string | undefined;
+  search: string;
+}
+
+export const PersonLink: React.FC<Props> = ({
+  person: {
+    slug,
+    sex,
+    name,
+    born,
+    died,
+    motherName,
+    fatherName,
+    mother,
+    father,
+  },
+  activeSlug,
+  search,
+}) => {
```

Comment:
Consider destructuring the `person` object directly in the function parameters. This way, you won't need to destructure it inside the function body. This is a code style suggestion and it's up to you to decide whether to apply it or not.